Makefile

```diff
@@ -34,8 +34,6 @@ lint:
 	mypy --strict samtranslator bin
 	# Linter performs static analysis to catch latent bugs
 	pylint --rcfile .pylintrc samtranslator
-	# Ensure templates adhere to JSON schema
-	bin/validate.sh
```

Comment:
Should not be removed. We need to ensure all templates going forward adhere to the schema.